gowin/pack.cc

```diff
@@ -636,7 +636,45 @@ static void set_net_constant(const Context *ctx, NetInfo *orig, NetInfo *constne
             CellInfo *uc = user.cell;
             if (ctx->verbose)
                 log_info("%s user %s\n", ctx->nameOf(orig), ctx->nameOf(uc));
-            if ((is_lut(ctx, uc) || is_lc(ctx, uc)) && (user.port.str(ctx).at(0) == 'I') && !constval) {
+
+            if ((is_lut(ctx, uc)) && (user.port.str(ctx).at(0) == 'I') && !constval) {
```

Comment:
I think the ` && !constval` term isn't actually needed here, as the same logic can be applied when an input is tied high as well as low

Reply:
Right, should I leave it in the function signature to be consistent with the other architectures although `constval` is not used?

Reply:
No, packing code is entirely arch-specific so there's no need to worry there